data/json/items/book/survival.json

```diff
@@ -132,6 +144,10 @@
     "color": "light_blue",
     "skill": "survival",
     "required_level": 5,
+    "proficiencies": [
+      { "proficiency": "prof_knapping", "time_factor": 0.5, "fail_factor": 0.5 },
+      { "proficiency": "prof_knapping_speed", "time_factor": 0.5, "fail_factor": 0.5 }
```

Comment:
This doesn't seem like a book that would have great clear directions.

Reply:
Changed to 0.8! Tell me if that number is more appropriate or if it would be better to rather have the modifier removed.

Reply:
Higher is better, that means it counts as being 80% of the way towards having the proficiency. Overall your other values could be close to 0.3 or so, and this should be in the 0.1ish range probably

Reply:
Uh oh, I did not see that PR, I will change my PRs accordingly.